src/test/java/org/springframework/data/jpa/repository/query/NamedQueryUnitTests.java

```diff
@@ -15,7 +15,8 @@
  */
 package org.springframework.data.jpa.repository.query;
 
-import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.ArgumentMatchers.any;
```

Comment:
Please make sure that static imports stay as wildcard imports.

Reply:
No longer relevant in the new push since this file wasn't touched.